packages/fast-usdc/test/exos/advancer.test.ts

```diff
@@ -357,17 +385,31 @@ test('will not advance same txHash:chainId evidence twice', async t => {
   mockPoolAccount.transferVResolver.resolve();
   await eventLoopIteration();
 
-  t.deepEqual(inspectLogs(0), [
-    'Advance transfer fulfilled',
-    '{"advanceAmount":{"brand":"[Alleged: USDC brand]","value":"[146999999n]"},"destination":{"chainId":"osmosis-1","encoding":"bech32","value":"osmo183dejcnmkka5dzcu9xw6mywq0p2m5peks28men"},"result":"[undefined]"}',
+  t.deepEqual(inspectLogs(), [
+    ['decoded EUD: osmo183dejcnmkka5dzcu9xw6mywq0p2m5peks28men'],
+    [
+      'Advance transfer fulfilled',
+      {
+        advanceAmount: { brand: usdc.brand, value: 146999999n },
+        destination: {
+          chainId: 'osmosis-1',
+          encoding: 'bech32',
+          value: 'osmo183dejcnmkka5dzcu9xw6mywq0p2m5peks28men',
+        },
+        result: undefined,
+      },
+    ],
   ]);
 
   // Second attempt
   void advancer.handleTransactionEvent(mockEvidence);
   await eventLoopIteration();
-  t.deepEqual(inspectLogs(1), [
-    'txHash already seen:',
-    '0xc81bc6105b60a234c7c50ac17816ebcd5561d366df8bf3be59ff387552761702',
+  const [, , ...remainingLogs] = inspectLogs();
```

Comment:
TIL array destructuring doesn't have to name elements

Reply:
I think I relearned that yesterday. Same with [assignment while destructuring](https://github.com/Agoric/agoric-sdk/blob/d369f0894104d51f30b188ec45aeedf65b5819d0/packages/fast-usdc/src/exos/settler.js#L154-L156.). JavaScript is fun 🙂